components/rmrk/Gallery/NewestList.vue

```diff
@@ -55,37 +55,44 @@ export default class NewestList extends mixins(PrefixMixin, AuthMixin) {
     return false
   }
 
-  mounted() {
-    setTimeout(async () => {
-      const queryVariables = {
-        limit: 10,
-        event: 'LIST',
-        and: {
-          meta_not_eq: '0',
-        },
-      }
-      if (this.isLogIn && this.passionList.length > 9) {
-        queryVariables.and.nft = {
-          issuer_in: this.passionList,
-        }
-      }
-      const result = await this.$apollo
-        .query<{
-          events: { meta; nft: { meta: { id; image } } }
-        }>({
-          query: lastNftListByEvent,
-          client: this.client,
-          variables: queryVariables,
-        })
-        .catch((e) => {
-          this.$consola.error(e)
-          return { data: null }
-        })
+  async fetch() {
+    this.fetchData()
+  }
```

Comment:
put `fetchData()` method here

Reply:
but wouldn't this make `hasPassionFeed` global, like if you disable it on landing page, it would also get disabled in all the other places as well like spotlight and gallery

Reply:
> but wouldn't this make `hasPassionFeed` global, like if you disable it on landing page, it would also get disabled in all the other places as well like spotlight and gallery

Personally I think this is the goal

Reply:
@roiLeo oh, then do we like remove toggles from everywhere and just control them through preferences, a single spot?